flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/MySqlValidator.java

```diff
@@ -50,6 +50,7 @@ public class MySqlValidator implements Validator {
 
     private static final String BINLOG_FORMAT_ROW = "ROW";
     private static final String BINLOG_FORMAT_IMAGE_FULL = "FULL";
```

Comment:
@ruanhang1993 
May I change the variable names `BINLOG_FORMAT_ROW`, `BINLOG_FORMAT_IMAGE_FULL` to `DEFAULT_BINLOG_FORMAT`, `DEFAULT_BINLOG_ROW_FORMAT`?

```suggestion
    private static final String DEFAULT_BINLOG_FORMAT = "ROW";
    private static final String DEFAULT_BINLOG_ROW_FORMAT = "FULL";
```


In mysql 8, default values for the `binlog_format` and `binlog_row_image` options are `ROW` and `full`, respectively
- https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_format
- https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_row_image

Reply:
Hi, @SML0127 .
I think we could not change these fields' names. These settings are checked by the `ROW` or `FULL` values.

Reply:
Thank you for your explanation, @ruanhang1993 
I undertand your responses. Do you have any other suggestions?